<launch>
  <group ns = "parloma">
    <param name="signs_topic" value="signs_topic" />
    <param name="serial_topic" value="serial_topic" />

    <node name="sign2command" pkg="hand_driver" type="sign2command.py" output = "screen">
      <param name="xml_hand" value="$(find hand_driver)/xml/robot_hand_Bulga.xml" />
      <param name="xml_signs" value="$(find hand_driver)/xml/signs2pose.xml" />
      <param name="xml_commands" value="$(find hand_driver)/xml/commands_list.xml" />
    </node>

    <node name="hand" pkg="serial_bridge" type="serial_node.py">
      <param name="port" value="/dev/tty.usbmodem1421" />
      <param name="baudrate" value="9600" />
    </node>
  </group>
</launch>
